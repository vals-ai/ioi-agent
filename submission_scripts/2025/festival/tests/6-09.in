version https://git-lfs.github.com/spec/v1
oid sha256:155aaff8bda03b5ffd3242fb8ff778936714cfb8bd916fcfe64e1b19e1ad8564
size 2395218
