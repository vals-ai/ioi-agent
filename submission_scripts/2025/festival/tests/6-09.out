version https://git-lfs.github.com/spec/v1
oid sha256:d9e49516dd3e999c274d08c1b1ad3c8e16b3dc2eacb567fe955fb4fb761dc86e
size 120
